Makefile

```diff
@@ -167,28 +168,28 @@ docker-push:
 	@docker push "$(DOCKER_IMAGE_REPO):$(DOCKER_IMAGE_TAG)"
 
 .PHONY: docs
-docs: ## Regenerates flags in docs for all thanos commands.
-docs: $(EMBEDMD) build
+docs: ## Regenerates flags in docs for all thanos commands localise links, ensure GitHub format.
+docs: $(MDOX) build
 	@echo ">> generating docs"
-	@EMBEDMD_BIN="$(EMBEDMD)" SED_BIN="$(SED)" THANOS_BIN="$(GOBIN)/thanos"  scripts/genflagdocs.sh
+	$(MDOX) fmt --links.localize.address-regex="https://thanos.io/.*" *.md $(MD_FILES_TO_FORMAT)
 	@echo ">> cleaning white noise"
 	@find . -type f -name "*.md" | SED_BIN="$(SED)" xargs scripts/cleanup-white-noise.sh
 
+
 .PHONY: check-docs
 check-docs: ## checks docs against discrepancy with flags, links, white noise.
-check-docs: $(EMBEDMD) build
-	@echo ">> checking docs generation"
-	@EMBEDMD_BIN="$(EMBEDMD)" SED_BIN="$(SED)" THANOS_BIN="$(GOBIN)/thanos" scripts/genflagdocs.sh check
-	@echo ">> checking links (DISABLED for now)"
+check-docs: $(MDOX) build
```

Comment:
Even though Thanos is getting built it isn't available in CI it seems and causes mdox-exec directives to fail.